packages/lit-html/src/lit-html.ts

```diff
@@ -394,9 +402,14 @@ export const render = (
       container.insertBefore(createMarker(), endNode),
       endNode,
       undefined,
-      options
+      options || {}
     );
   }
+  // Pass top-level part connection status to AsyncDirectives via options
+  (part.options as PrivateRenderOptions) = {
```

Comment:
Only copy this if there's no isConnected flag?

Reply:
Moved the copy into the one-time part creation step.